CryptoPkg/Library/BaseCryptLib/Pk/CryptPkcs7VerifyEku.c

```diff
@@ -451,7 +451,7 @@ VerifyEKUsInPkcs7Signature (
   //
   // Create the PKCS7 object.
   //
-  Pkcs7 = d2i_PKCS7(NULL, (CONST CHAR8 **)&Temp, (INT32)SignedDataSize);
+  Pkcs7 = d2i_PKCS7(NULL, (const unsigned char **) &Temp, (INT32)SignedDataSize);
```

Comment:
Shouldn't this just be CONST UINT8, since we're just passing a byte stream?

Reply:
Nevermind. It seems you're using the established practice in TianoCore. Approved.